### PR TITLE
add new page udp

### PR DESCRIPTION
**Pull request by elmalakomar**
## Pull Request Type

- [ ] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensitive content has been committed

### CODE REVIEW

File: docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+id: udp
+title: 'Envoy Use Case: UDP'
+sidebar_label: 'Use Case: UDP'
+---
+
+Envoy allows to accept UDP requests.
+
+The Mia-Platform Console helps to configure the Envoy API Gateway with the advanced section.
```
Comment (sfontana9) [inline docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md:9]: I would say `using` the advanced section.

File: docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - name: envoy.filters.udp_listener.udp_proxy 
+      typed_config:
+        "@type": type.googleapis.com/envoy.extensions.filters.udp.udp_proxy.v3.UdpProxyConfig
+        stat_prefix: udp_stats  
+        cluster: udp_cluster
+        idle_timeout: 60s    
+```
+
+This configuration creates a listener named `udp` that accepts and proxies UDP traffic. 
+The listener listen on the port 33333 for incoming UDP packets accepting from any address.
```
Comment (sfontana9) [inline docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md:34]: I would change "The listener listen on the port" into "The listener monitors the port"

File: docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+As listener configuration we set a maximum size for incoming packets to 9000 bytes.
```
Comment (sfontana9) [inline docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md:36]: I would say "In this configuration we set". Is this something we use as an example, or are we recommending/suggesting the reader to do it?

File: docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+We also define a proxy filter that indicates `udp_cluster` as cluster to which forward the traffic and sets a timeout of 60 seconds.
```
Comment (sfontana9) [inline docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md:38]: I would change the phrase into "... indicates `udp_cluster` as the cluster to forward the traffic to. Then sets a timeout ..."

File: docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md (hunk 3 above)
Comment (elmalakomar) [reply docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md]: This configuration is as example. I changed the sentence:

```
In this configuration we set a maximum size for incoming packets to 9000 bytes. If not set, the default is 1500 bytes. 
```

Comment (sfontana9) [reply docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md]: Great!

File: docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+---
+id: udp
+title: 'Envoy Use Case: UDP'
+sidebar_label: 'Use Case: UDP'
+---
+
+Envoy allows to accept UDP requests.
+
+The Mia-Platform Console helps to configure the Envoy API Gateway using the advanced section.
```
Comment (akelity) [inline docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md:9]: Can we clarify better where the advanced section is located?

File: docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      max_rx_datagram_size: 9000
+  listener_filters: 
+    - name: envoy.filters.udp_listener.udp_proxy 
+      typed_config:
+        "@type": type.googleapis.com/envoy.extensions.filters.udp.udp_proxy.v3.UdpProxyConfig
+        stat_prefix: udp_stats  
+        cluster: udp_cluster
+        idle_timeout: 60s    
+```
+
```
Comment (akelity) [inline docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md:32]: We should probably clarify better what each field in the JSON above represents (to make it clear to the user what to input)

File: docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    cluster_name: udp_cluster
+    endpoints:
+      - lb_endpoints:
+          - endpoint:
+              address:
+                socket_address:
+                  address: udp-server
+                  port_value: 33333   
+```
+
```
Comment (akelity) [inline docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md:60]: Here as well, can we specify what the fields in the JSON are?

**Review by akelity — CHANGES_REQUESTED** [review]

**Review by akelity — APPROVED** [review]

**Pull request merged**